### ISSUE TITLE
Standard-minifier-css error?

### ISSUE THREAD

**Issue by Themandunord** [meteor/meteor#9458]
Hi guys !

I want to import the less file from ant design.

So i put a *.less file in my project and write : `@import '{}/node_modules/antd/dist/antd.less';` 
(tried with `@import '../../../node_modules/antd/dist/antd.less';`).

And put `@primary-color  : #d70b52;` to change the color.

Well, it's a stantard code.

But when I start my app, i have : 
```
=> Errors prevented startup:

   While minifying app stylesheet:
   app/client/configs/style/ant.theme.less.css:34:3: missing '}'
```

I look at issues on gitub / forums / ... but I don't a clear resolution of this error.
I see a comment from @hwillson on [Github](https://github.com/meteor/meteor/issues/6846#issuecomment-336233084).

And when I did `meteor remove standard-minifier-css`, my app launched and works.
The color has been changed, so it's perfect.

But, why ?
I need the `standard-minifier-css` in production, isn't it ?

If I add `juliancwirko:postcss` I have the same problem...

So guys, you got a solution for my issue ? 
Btw, it is an issue with my code you think ? Or with my configuration ? 

Thanks :slight_smile:

Meteor : 1.6

**Comment by hwillson** [comment]
Hi @Themandunord - any chance you could put together a small runnable reproduction that demonstrates this issue? See [Reporting a bug in Meteor](https://github.com/meteor/meteor/blob/devel/Contributing.md#reporting-bug) for more info. Thanks!

**Comment by Themandunord** [comment]
Hi @hwillson I will provide you a reproduction example :)
It's necessary to have a very little app ? I have a kickstarter that have the problem if you want now.

PS: your link is a 404 ;)

**Comment by hwillson** [comment]
Thanks @Themandunord - I've fixed the link.

**Comment by hwillson** [comment]
And to answer your question - your kickstarter should be okay; if you can provide a link, I'll take a look.

**Comment by Themandunord** [comment]
Here the link to my repo : https://github.com/Themandunord/Mantra-Kickstarter-2

Add : `meteor add standard-minifier-css` or `meteor add juliancwirko:postcss` to see the bug during the build time.

If it doesn't work, I can create a sample reproduction.
Thanks.

**Comment by Themandunord** [comment]
Hum ok... I find the bug !

```
// IE10+ doesn't honor `<meta name="viewport">` in some cases.
@at-root {
  @-ms-viewport { width: device-width; }
}
```
[Here](https://github.com/ant-design/ant-design/blob/master/components/style/core/base.less#L53)

When I remove this code from the base.less file. I can launch my app without error.

So, it's a related to a meteor bug or a antd bug ? 

Thanks for watching that @hwillson 👍

**Comment by hwillson** [comment]
This is a Meteor bug, caused by the old CSS parsing library that's being used. The good news is that this has already been fixed (PR https://github.com/meteor/meteor/pull/9263). The fix will be coming in Meteor 1.6.1. Thanks!

**Comment by Themandunord** [comment]
Oh thanks @hwillson for this PR 👍 
I will test the 1.6.1.beta release so !
